content/client-server-api/modules/reference_relations.md

```diff
@@ -0,0 +1,51 @@
+---
+type: module
+---
+
+### Reference relations
+
+{{% added-in v="1.4" %}}
+
+Generically referencing another event can be done with a `rel_type` of `m.reference`
+as a form of [relationship](#forming-relationships-between-events). There is no
+implied meaning behind the reference, and is usually context-dependent. Such an
+example is the [key verification framework](#key-verification-framework) which uses
+reference relations to associate distinct events with a specific verification attempt.
+
+When bundled, a reference relations appear as an object with a `chunk` field to
```

Comment:
s/bundled/aggregated/ ? with a link to https://spec.matrix.org/v1.3/client-server-api/#aggregations ?

Reply:
Also s/field/property/ per https://github.com/matrix-org/matrix-spec/blob/main/meta/documentation_style.rst#general.

```suggestion
The aggregation format of `m.reference` relations consists of a single `chunk` property to
```

Reply:
> s/bundled/aggregated/ ? with a link to https://spec.matrix.org/v1.3/client-server-api/#aggregations ?

The MSC says bundled and I was hoping that would magically fit in our spec language 😭